connexion/apis/aiohttp_api.py

```diff
@@ -278,14 +279,17 @@ def get_request(cls, req):
         if req.body_exists:
             body = yield from req.read()
 
+        data = yield from req.post()
```

Comment:
As far as I can tell, this will never work.
If you post the request, whatever that is because important part is to have something in that, so the condition `req_body_exists` evaluates to `True`, you are reading the entire `payload` from `aiohttp.web.Request` in line **280** (`yield from req.read()`). Later on you are attempting to read the `payload` yet again via `req.post()`. That's why it fails.

What might have be needed is here is to distinguish between multipart requests and ordinary requests.
I assume that multipart requests might be processed in a way that we extract files and normal fields into separate variables and push'em into `ConnexionRequest`.

Reply:
Good catch! 
Indeed, you can't read `aiohttp.web.Request` payload twice, since [it's a](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/streams.py#L93) `StreamReader` object and both methods read from it.
Calling different readers based on the content type sounds like a solution.

Reply:
@prawn-cake  Give it a whirl then :)

Reply:
@prawn-cake I will give it a try on my own. We're interested in having it fixed as soon as possible.

Reply:
Thanks @kornicameister !
Sorry for not being very active here. I'll check the changes